src/yw_funcs.cpp

```diff
@@ -531,7 +531,7 @@ int yw_InitLevelNet(_NC_STACK_ypaworld *yw)
     }
 
     yw_InitLevelNet__sub0(yw);
-    if ( !yw_ParseWorldIni(yw, "env:world.ini") && !yw_ParseWorldIni(yw, "data:world.ini"))
+    if ( !yw_ParseWorldIni(yw, "data:world.ini") && !yw_ParseWorldIni(yw, "env:world.ini"))
```

Comment:
What will happen when you will try to play expansion where world.ini in env dir?

Reply:
There would be an error message :D I have not inspected expansion so this pull request can closed. Code blocks project is not compiling without C++11 options however.